client/blocks/login/social.jsx

```diff
@@ -28,13 +28,15 @@ import { InfoNotice } from 'blocks/global-notice';
 import { login } from 'lib/paths';
 
 function isSafari() {
-	return typeof window !== 'undefined' && window.navigator.userAgent.match( /safari/i );
+	return (
+		typeof window !== 'undefined' && /^(?!.*chrome).*safari/i.test( window.navigator.userAgent )
```

Comment:
Why chrome is in the regex?
Safari has something like:
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Safari/604.1.38`

Reply:
oops, I missed the `?!`, sorry =)

Reply:
Try `navigator.userAgent` on chrome you'll see ;)
In my case I have this:
`"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36"`

Reply:
Yeah, I haven't noticed the lookahead, sorry =)